utils.py

```diff
@@ -81,3 +84,11 @@ def blockchain_transactions(logger, node, wallet_address, limit, after=None):
 def apply_merchant_rate(amount, rate):
     amount_nzd = (amount / float(1 + rate))
     return amount_nzd
+
+def is_email(email):
+    return re.match("[^@]+@[^@]+\.[^@]+", email)
+
+def generate_random_password(length):
```

Comment:
change this to use a crypto secure source of randomness: https://docs.python.org/3/library/secrets.html#recipes-and-best-practices

Reply:
im reading this and from what i could tell is to replace random and use secrets instead.

done.